.github/scripts/shared/circle-artifacts.ts

```diff
@@ -0,0 +1,181 @@
+import fs from 'fs';
+
+// Set OWNER and REPOSITORY based on the environment
+const OWNER = process.env.CIRCLE_PROJECT_USERNAME || process.env.OWNER;
+const REPOSITORY = process.env.CIRCLE_PROJECT_REPONAME || process.env.REPOSITORY;
```

Comment:
For `yarn download-builds` to work, this needs some defaults
```suggestion
const OWNER =
  process.env.CIRCLE_PROJECT_USERNAME || process.env.OWNER || 'MetaMask';
const REPOSITORY =
  process.env.CIRCLE_PROJECT_REPONAME ||
  process.env.REPOSITORY ||
  'metamask-extension';
```